src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -1193,12 +1195,12 @@ FixedwingPositionControl::control_auto_loiter(const hrt_abstime &now, const Vect
 		prev_wp(1) = pos_sp_curr.lon;
 	}
 
-	float mission_airspeed = _param_fw_airspd_trim.get();
+	float airspeed_sp = -1.f;
 
 	if (PX4_ISFINITE(pos_sp_curr.cruising_speed) &&
-	    pos_sp_curr.cruising_speed > 0.1f) {
+	    pos_sp_curr.cruising_speed > FLT_EPSILON) {
```

Comment:
@sfuhrer In theory you should probably also check if the triplet is valid.

Reply:
we use the triplet all over the place and only sometimes check for validity in the FW position controller..wonder how important it can be - when can the triplet be invalid?